src/components/Repositories/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useEffect } from 'react';
+import axios from "axios";
+import UserListing from '../UserListing';
+import RepoListing from '../RepoListing';
+
+
+const url = "https://api.github.com"
```

Comment:
having this const is nice, so if the API ever changes address you only have to change this one variable :100: